go get error

I run the `go get github.com/airbusgeo/godal`, and got the follow error.
```bash
....
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1802:34: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1711:34: error: use of undeclared identifier 'oCC'
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1713:32: error: use of undeclared identifier 'oCC'
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1717:53: error: use of undeclared identifier 'oCC'
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1719:51: error: use of undeclared identifier 'oCC'
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1880:24: error: function definition does not declare parameters
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1883:15: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1885:15: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1893:24: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
/usr/local/Cellar/gdal/3.2.0_1/include/ogr_geometry.h:1914:49: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
....
fatal error: too many errors emitted, stopping now [-ferror-limit=]
```

--- a/.github/workflows/osx.yml
+++ b/.github/workflows/osx.yml
@@ -1,0 +1,23 @@
+name: osx-build
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  osx:
+    runs-on: macOS-latest
+    name: macOS test
+    steps:
+      - name: install gdal
+        run: brew install pkg-config gdal proj geos
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Tests
+        run: go test . -race

--- a/godal.go
+++ b/godal.go
@@ -19,6 +19,7 @@ package godal
 #include <stdlib.h>
 
 #cgo pkg-config: gdal
+#cgo CXXFLAGS: -std=c++11
 #cgo LDFLAGS: -ldl
 */
 import "C"

